bundle/manifests/search-v2-operator.clusterserviceversion.yaml

```diff
@@ -0,0 +1,217 @@
+apiVersion: operators.coreos.com/v1alpha1
+kind: ClusterServiceVersion
+metadata:
+  annotations:
+    alm-examples: |-
+      [
+        {
+          "apiVersion": "cache.open-cluster-management.io/v1",
+          "kind": "OCMSearch",
+          "metadata": {
+            "name": "search-v2-operator",
+            "namespace": "open-cluster-management"
+          },
+          "spec": null
+        }
+      ]
+    capabilities: Basic Install
+    operators.operatorframework.io/builder: operator-sdk-v1.15.0
+    operators.operatorframework.io/project_layout: go.kubebuilder.io/v3
+  name: search-v2-operator.v2.0.0
```

Comment:
What does `v2.0.0` means here?  
Our operator name creates some confusion. This is `v1` of `search-v2-operator`. 😂

Reply:
Updated to 0.0.1